libexec/python/docker/api.py

```diff
@@ -47,24 +52,46 @@ def create_runscript(cmd,base_dir):
     return output_file
 
 
-def get_token(repo_name,namespace="library",scope="repository",permission="pull"):
+def get_token(registry,repo_name,namespace,scope="repository",permission="pull",auth=None):
     '''get_token will use version 2.0 of Docker's service to return a token with given permission and scope - this
     function does work, but the token doesn't seem to work when used with other functions below for authentication
```

Comment:
+1. Also - I'll take care of updating the online docs to give details about using the command line token for import once this is merged. I'll ping you with the link to check over for a second set of eyes.